tracim/tracim/model/data.py

```diff
@@ -54,11 +51,18 @@ class Workspace(DeclarativeBase):
     label   = Column(Unicode(1024), unique=False, nullable=False, default='')
     description = Column(Text(), unique=False, nullable=False, default='')
 
-    created = Column(DateTime, unique=False, nullable=False)
-    updated = Column(DateTime, unique=False, nullable=False)
+    created = Column(DateTime, unique=False, nullable=False, default=datetime.utcnow)
+    updated = Column(DateTime, unique=False, nullable=False, default=datetime.utcnow)
 
     is_deleted = Column(Boolean, unique=False, nullable=False, default=False)
 
+    revisions = relationship("ContentRevisionRO")
```

Comment:
Before content view deletion, ``contents`` was linked by back reference. Now i compute it from a relation with revisions table. Maybe it's possible to use here a relation with content, with manually builded join (see http://docs.sqlalchemy.org/en/latest/orm/relationship_api.html#sqlalchemy.orm.relationship.params.primaryjoin)